cpp/plugins/cucim.kit.cuslide/cmake/deps/libopenjpeg.cmake

```diff
@@ -38,7 +38,6 @@ if (NOT TARGET deps::libopenjpeg)
     ###########################################################################
     # Build liblcms2 with the source in libopenjpeg
     ###########################################################################
-    message(" ##GG ${deps-libopenjpeg_SOURCE_DIR}")
```

Comment:
The message was added during debugging (to see which path is used) so removed here.

Since `deps-libopenjpeg_SOURCE_DIR` would be populated by `FetchContent_GetProperties()` above, we don't need to check its existence.